### PR TITLE
page : Result page Added , p1 features for quiz

### PR DESCRIPTION
**Pull request by manojsarna**
Result Page Added

P1- Features

Timer for Quiz

Search Quiz

Quit Quiz

Authentication

### CODE REVIEW

**Review by b0llu — APPROVED** [review]
Great work on adding p1 components in quiz! Everything is looking good nothing to suggest

File: lib/Components/Result/result.css [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+    background: var(--COLOR-HEADER-FONT-DARK);
+}
+
+.sm-quiz-nav-btn {
+    display: flex;
+    justify-content: space-between;
+}
+
+.btn-end-quiz {
+    text-decoration: none;
```
Comment (bhtibrewal) [inline lib/Components/Result/result.css:108]: Suggestion: do text-decoration:none in reset so u don't have to do repeatedly

File: lib/Components/Result/result.html [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+
+<body>
+    <header class="sm-header">
+        <nav class="sm-nav-new">
+            <!-- LOGO -->
+            <a href="/" class="link-text-dec" title="Go To Home">
+                <div id="sc-logo" class=" sm-nav-logo flex-row-wrap sm-logo">
+                    <span class="sm-header-icon">
+                        <i class="fas fa-clone"></i>
+                    </span>
```
Comment (pankaj-prog) [inline lib/Components/Result/result.html:21]: You can directly style the icon instead of wrapping it in another element

**Review by pankaj-prog — APPROVED** [review]
You can reduce the sizes of your components. Just a UI suggestion.

File: lib/Components/Result/result.css [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+
+    h2,
+    .h2 {
+        font-size: 2rem;
+    }
+
+    .sm-main-ques {
+        padding: 1.5rem;
+    }
+}
```
Comment (bhtibrewal) [inline lib/Components/Result/result.css:154]: Confusion: do you need to change the font size like this in the media query? you have changed the root font size I think that should be enough

File: lib/Components/Result/result.html [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+</head>
+
+<body>
+    <header class="sm-header">
+        <nav class="sm-nav-new">
+            <!-- LOGO -->
+            <a href="/" class="link-text-dec" title="Go To Home">
+                <div id="sc-logo" class=" sm-nav-logo flex-row-wrap sm-logo">
+                    <span class="sm-header-icon">
+                        <i class="fas fa-clone"></i>
```
Comment (bhtibrewal) [inline lib/Components/Result/result.html:20]: Change: the favicon logo with the title is different than this one

**Review by bhtibrewal — APPROVED** [review]
Mostly everything looks good , i have suggested just one change and other suggestion, after that good to merge

File: lib/Components/Result/result.css [hunk 5]
```diff
@@ -0,0 +1,154 @@
...
+    background: var(--COLOR-HEADER-FONT-DARK);
+    color: var(--COLOR-BLACK);
+    border: 1px solid var(--COLOR-HEADER-FONT-DARK);
+}
+
+/* MAIN ENDS */
+
+/* MEDIA QUERIES */
+
+@media only screen and (max-width:768px) {
```
Comment (sneha-amruth) [inline lib/Components/Result/result.css:127]: Put your breakpoints in variables and use it across your files

File: lib/Components/Auth/auth.css [hunk 6]
```diff
@@ -2,67 +2,90 @@
...
-        font-size: 45%;
-    }
+.sm-auth-forget {
+    align-self: flex-end;
+    background: none;
+    border: none;
+    cursor: pointer;
+    outline: none;
+    font-size: 1.2rem;
+    color: var(--COLOR-HEADER-FONT-DARK);
```
Comment (sneha-amruth) [inline lib/Components/Auth/auth.css:58]: You can skip appending color to your color variables since its redundant.

**Review by sneha-amruth — COMMENTED** [review]
Suggested few minor changes

**Pull request merged**